utils/config-utils.js

```diff
@@ -11,6 +11,7 @@ export function initCuttleGlobals(app) {
     version,
     // Expose app for debugging/testing in lower envs
     app: test ? app : null,
```

Comment:
Convert this to a spread operator with an object with both values.

Reply:
Removed the socket from the window altogether so that vitest can import this file without exploding